pkg/types/orderbook.go

```diff
@@ -239,6 +239,12 @@ func (sb *StreamOrderBook) BindStream(stream Stream) {
 	})
 }
 
+func (sb *StreamOrderBook) BindUpdateMetrics(updateMetrics func(book *StreamOrderBook, update SliceOrderBook)) {
```

Comment:
This function does not seem related to metrics? Is there a better method name for this?

Reply:
or maybe just BindUpdate()

Reply:
Ok, `BindUpdate` works for me.

Reply:
Done.